app/src/main/res/xml/backup_rules.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
```

Comment:
we don't need the copyright  in xml. I had a small look into it because they were automatically getting inserted for me too but I couldn't find a satisfactory explanation